src/sentry/similarity/features.py

```diff
@@ -138,7 +138,7 @@ def record(self, events):
             scope,
             key,
             items,
-            timestamp=to_timestamp(event.datetime),
+            timestamp=int(to_timestamp(event.datetime)),
```

Comment:
Not sure how we hadn't hit this yet (maybe we truncate timestamps in the pipeline somewhere?) but using the current timestamp with higher precision caused this to error.